### PR TITLE
Test only with currently supported versions of Python

### PR DESCRIPTION
**Pull request by jdevera**

### CODE REVIEW

**Comment by jdevera** [comment]
Integration fails due to one of the dependencies pulling a version of yarl that is not compatible with Python 3.5:

    RuntimeError: yarl 1.4+ requires Python 3.6+

This is a problem of moving, no-pinned indirect dependencies. Users will find this when installing the latest released version on Python 3.5

It is however unrelated to this PR.

**Comment by pkopac** [comment]
Thanks for the PR, @jdevera . There are a few things, though, before we can merge and release it:

This needs a version update (to1.3.0):
https://github.com/chartmogul/chartmogul-python/blob/master/chartmogul/__init__.py#L32

...and README update:
>This library requires Python 2.7 or 3.3 to 3.6.

The build also has to be fixed before merging.

**Comment by jdevera** [comment]
Hi, thanks for the feedback.
I will update the README.
I can also add the version change, but would like it done in the PR directly?
However, the other build issue is present in master and is unrelated to this PR, I doubt it should be here.

**Comment by jdevera** [comment]
To fix, this seems enough, in `setup.py`:
```diff
 test_requirements = [
+    'yarl<1.4',
     'mock>=1.0.1',
```

**Comment by jdevera** [comment]
> To fix, this seems enough, in `setup.py`:
> 
> ```diff
>  test_requirements = [
> +    'yarl<1.4',
>      'mock>=1.0.1',
> ```
> 
> Shall I?

Actually this breaks then for 2.X

**Comment by pkopac** [comment]
@jdevera thanks! We need to take look at it in detail to resolve this, but don't have time till next week. We might need to create a legacy branch like we did with our PHP client that will conserve 2.* support.
I added the task to backlog.

It seems we can reasonably stop supporting new development [compatible with 2.* Python from January 2020](https://wiki.python.org/moin/Python2orPython3)

**Comment by jdevera** [comment]
Found the problem, it was vcrpy https://github.com/kevin1024/vcrpy/issues/485

I added a workaround that you can either remove or leave when vcrpy is fixed.

That should make it all green.

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 ## Installation
 
-This library requires Python 2.7 or 3.3 to 3.6.
+This library requires Python 2.7 or 3.6 to 3.8.
```
Comment (hassansin) [inline README.md]: Should be 3.5 5o 3.8

```suggestion
This library requires Python 2.7 or 3.5 to 3.8.
```

Comment (jdevera) [reply README.md]: Applied. Thanks.

**Pull request merged**